ecl/hql/hqlexpr.hpp

```diff
@@ -1276,20 +1278,24 @@ struct OwnedHqlExprItem : public CInterface
 };
 
 extern HQL_API const char *getOpString(node_operator op);
-extern HQL_API IHqlExpression *createValue(node_operator op);
-extern HQL_API IHqlExpression *createValue(node_operator op, IHqlExpression *p1, IHqlExpression *p2);
 extern HQL_API IHqlExpression *createOpenValue(node_operator op, ITypeInfo *type);
 extern HQL_API IHqlExpression *createOpenNamedValue(node_operator op, ITypeInfo *type, IIdAtom * id);
 extern HQL_API IHqlExpression *createNamedValue(node_operator op, ITypeInfo *type, IIdAtom * id, HqlExprArray & args);
 extern HQL_API IHqlExpression *createId(IIdAtom * id);
 
+extern HQL_API IHqlExpression *createValue(node_operator op);
+extern HQL_API IHqlExpression *createValue(node_operator op, IHqlExpression *p1, IHqlExpression *p2);
+extern HQL_API IHqlExpression* createValue(node_operator op, HqlExprArray& operands);
+extern HQL_API IHqlExpression *createValue(node_operator op, IHqlExpression *p1);
```

Comment:
picky, but feels more natural for this to be two lines higher...

Reply:
I agree, moved in subsequent commit.